.github/workflows/lint.yaml

```diff
@@ -11,6 +11,11 @@ jobs:
     steps:
       - name: Checkout code
         uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+      - name: Update Go
+        uses: actions/setup-go@v4
+        with:
+          go-version: '>=1.21.0'
+          cache: false
```

Comment:
I agree that we should cover the *new* code with linting. I don't think it's worth to lint also with older Go because the non-slog code isn't that complex and going away at some point.

Reply:
i cant tell if you are agreeing with me or want a change?

Reply:
I agree with you.